app/controllers/items_controller.rb

```diff
@@ -45,7 +45,7 @@ def create
   def search
     if !params[:search].empty?
       @phrase = params[:search]
-      @items = Item.all.where("lower(name) LIKE :phrase", phrase: @phrase) 
+      @items = Item.all.where("lower(name) LIKE :phrase OR lower(category) LIKE :phrase OR lower(description) LIKE :phrase", phrase: @phrase.downcase) 
```

Comment:
2 questions:

1. Aren't we at risk of SQL injections if we directly place these in here? I think what we want here is put placeholders to avoid this issue: `"lower(name) ? ", LIKE :phrase`
2. What is phrase: @phrase.downcase doing here? I tried to find documentation on this and couldn't figure it out... If what it does is convert phrase to lowercase, then we probably don't need the `lower()` on each attribute.